### PR TITLE
Post deploy smoke tests

### PR DESCRIPTION
**Pull request by LewisDaleUK**

### CODE REVIEW

File: scripts/run-smoke-tests.sh [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+if [ -z "$WORKSPACE" ]
+then
+  echo "WORKSPACE is not set"
+  exit 1
+fi
+URL=$(aws elbv2 describe-load-balancers --names cjse-${WORKSPACE}-bichard-7-lb | jq '.LoadBalancers[0].DNSName' -r)
```
Comment (bjpirt) [inline scripts/run-smoke-tests.sh:8]: Might not need to use jq for this - I htink the AWS CLI lets you pass the --query parameter to do a similar thing

Not an issue, but I thought it was a nice feature!

**Review by bjpirt — APPROVED** [review]
Nice bit of shell scripting!

File: scripts/run-smoke-tests.sh (hunk 1 above)
Comment (bjpirt) [reply scripts/run-smoke-tests.sh:8]: Was curious so just checked - you can do this:

`aws elbv2 describe-load-balancers --names cjse-current-dev-bichard-7-lb --query 'LoadBalancers[0].DNSName' --output text`

Comment (LewisDaleUK) [reply scripts/run-smoke-tests.sh:8]: Ah cool, wasn't aware of that - thanks

**Pull request merged**